src/libxtp/gwbse/bse_operator.cc

```diff
@@ -77,33 +51,21 @@ Eigen::RowVectorXd BSE_OPERATOR<cqp, cx, cd, cd2>::row(int index) const {
 }
 
 template <int cqp, int cx, int cd, int cd2>
-Eigen::RowVectorXd BSE_OPERATOR<cqp, cx, cd, cd2>::Hx_row(int index) const {
-  int thread_id = OPENMP::getThreadId();
-  if (_Hx_cache[thread_id].hasValue(index)) {
-    return _Hx_cache[thread_id].getValue(index);
-  }
-  int auxsize = _Mmn.auxsize();
-  vc2index vc = vc2index(0, 0, _bse_ctotal);
+Eigen::MatrixXd BSE_OPERATOR<cqp, cx, cd, cd2>::OperatorBlock(int row,
```

Comment:
I'm a bit confused here. Does this means that we can use the Block only for an operator that contains only Hx ?

Reply:
Ok I guess that you can have both, and then you compute the 'rows' product first and then you add up the block product. Right ?

Reply:
yes